item.go

```diff
@@ -17,18 +16,16 @@ const ItemVersion uint64 = 1
 
```

Comment:
should we bump `ItemVersion` to 2?

Reply:
good call, will mark this PR as breaking too